content/operate/rc/subscriptions/maintenance/_index.md

```diff
@@ -33,19 +43,21 @@ Redis will notify users by email when maintenance starts and ends. For more deta
 
 During maintenance, your database will be operational, but you may notice some latency when connecting to your databases. 
 
-Your application may also disconnect from your database for a few seconds. Most Redis clients are set to refresh their DNS address when they reconnect to the database, and you will not be required to perform any further action. If you encounter connectivity problems for more than a minute during maintenance, please refresh your DNS entries.
+Your application may also disconnect from your database for a few seconds. Most Redis clients are set to refresh their DNS address when they reconnect to the database, and you will not be required to perform any further action. If you encounter connectivity problems for more than a minute during maintenance, please refresh your DNS entries. See [Resilient applications with Redis Cloud]({{< relref "/operate/rc/resilient-apps" >}}) for more info.
 
 ### Urgent maintenance
 
 Urgent maintenance refers to any activity that could affect service and cannot wait for scheduling. This includes applying urgent security patches.
 
 Redis can perform urgent maintenance at any time, even if you have set a manual maintenance window or have temporarily [skipped maintenance]({{< relref "/operate/rc/subscriptions/maintenance/set-maintenance-windows#skip-maintenance-temporarily" >}}). Redis will notify you by email when urgent maintenance starts and ends.
 
-## Notifications
+### Major upgrades
 
-Redis will notify you by email when maintenance starts and ends. If Redis needs an action from you to start maintenance, Redis will notify you with a reasonable amount of time before planned maintenance.
+We won't schedule major upgrades or upgrades that might include breaking changes until you opt-in to those changes. You can opt-in to specific major changes on the [subscription page]({{< relref "/operate/rc/subscriptions/view-pro-subscription" >}}).
```

Comment:
Suggestion - replace "You can opt-in to specific major changes on the [subscription page]" to "You can control your Redis major versions on the [subscription page]"

Reply:
```suggestion
We won't schedule major upgrades or upgrades that might include breaking changes until you opt-in to those changes. You can control your Redis major version on the [subscription page]({{< relref "/operate/rc/subscriptions/view-pro-subscription" >}}).
```